ledger-core/virtual/object/request_table.go

```diff
@@ -13,20 +13,22 @@ import (
 )
 
 type PendingTable struct {
-	lists map[contract.InterferenceFlag]*PendingList
+	lists []*PendingList
```

Comment:
there is even no need for a slice - better to use an array

Reply:
good point, will fix:
```
goos: darwin
goarch: amd64
pkg: github.com/insolar/assured-ledger/ledger-core/virtual/object
BenchmarkPendingTable-12    	 7344967	       154 ns/op	     160 B/op	       4 allocs/op
BenchmarkWorkingTable-12    	 5528679	       192 ns/op	     208 B/op	       5 allocs/op
```